test/unit/number/format-properties.js

```diff
@@ -77,11 +78,11 @@ QUnit.test( "should return symbolMap", function( assert ) {
 	});
 
 	assert.deepEqual( properties( "0", fa )[ 18 ], {
-		"%": "٪",
-		"+": "\u200e+\u200e",
-		",": "٬",
-		"-": "\u200e−",
 		".": "٫",
+		",": "٬",
+		"%": "٪",
+		"+": "‎+",
+		"-": "‎−",
```

Comment:
I merged the PR, but realized this test change is using the invisible character itself instead of the explicit `\u200e`. Can we revert that in a new PR? Thanks

Reply:
The explicit character makes reviewing (and understanding) the test easier.

Reply:
@rxaviers sure! I'll create a new PR to fix the issue #544 and I'll rever it there

Reply:
Thank you 🙇